src/components/Footer/Footer.jsx

```diff
@@ -1,7 +1,48 @@
 import React from "react";
+import { logo_image } from "assets/";
+import "./Footer.scss";
 
 const Footer = () => {
-  return null;
+    return (
+        <footer className="flex flex-col w-full">
+            <div id="top" className="flex w-full">
+                <nav className="relative flex w-4/12 p-6">
+                    <ul>
+                        <li className="text-left p-2">
+                            <a href="/services">Services</a>
+                        </li>
+                        <li className="text-left p-2">
+                            <a href="/testimonials">Testimonials</a>
+                        </li>
+                        <li className="text-left p-2">
+                            <a href="/contact">Contact Us</a>
+                        </li>
+                    </ul>
+                </nav>
+                <div className="flex w-4/12">
+                    <img
+                        src={logo_image}
+                        alt="Clarity Detailing Solutions Logo"
+                        className="m-auto w-1/4"
+                    />
+                </div>
+                <div className="relative p-6 flex flex-col w-4/12">
+                    <p className="text-right w-full p-1">123 Main Street</p>
+                    <p className="text-right w-full p-1">New York, NY 10030</p>
+                    <p className="text-right w-full p-1">905-739-2859</p>
+                    <a className="text-right w-full p-1" href="mailto:contact@claritydetailingsolutions.com">
```

Comment:
If you try this email link at small screen sizes it's breaking badly, I think because it's one long string. 

I'm using `work-break: break-word" to break by characters on other components, should we consider making this a global style?

Reply:
I don't think this should be global unless it's used quite commonly.

For now it should probably remain a case by case thing.

Resolved, though.